ADR/0039-send-cloud-events.md

```diff
@@ -0,0 +1,34 @@
+# 40. Konflux should send cloud events all system events.
+
+Date: 2024-09-24
+
+## Status
+
+Proposed
+
+## Context
+
+Konflux had made the architectural decision to not use cloud events. However, that does not mean that
+Konflux should not emit cloud events.
+
+Emitting cloud events would allow Konflux users to easily track what is happening in the system. In addition,
+they can use these cloud events to create their own product-specific infrastructure to support their build
```

Comment:
Can we use a different term other than "product-specific" throughout this document? AFAIK, Konflux is not aware of what a product is. Maybe use "application-specific"?

Reply:
or even "team-specific" or "org-specific"